pkgs/tools/misc/topgrade/default.nix

```diff
@@ -18,7 +18,10 @@ rustPlatform.buildRustPackage rec {
     sha256 = "sha256-9zP+rWhaK4fC2Qhd0oq9WVvCkvryooYo09k7016Rbxw=";
   };
 
-  cargoSha256 = "sha256-otn0XvZ0wufD+4mCGSM0hevKM+wWSvFVCKtTu/5m1uA=";
+  cargoPatches = lib.optionals stdenv.isDarwin [ ./darwin-cargo-lock.patch ];
```

Comment:
Please make this patch unconditional. Also we should upstream this rather than vendoring it.